sale_contact_type/readme/CONTRIBUTORS.rst

```diff
@@ -0,0 +1 @@
+* Alexandre Díaz <alexandre.diaz@tecnativa.com>
```

Comment:
Put me as designer and extend reviewer ;)

And put also with Tecnativa URL and 2 levels.